doc/usage/al2023.md

```diff
@@ -67,3 +68,73 @@ One can pass the NVIDIA driver major version using the following:
 make k8s=1.29 os_distro=al2023 enable_accelerator=nvidia enable_efa=true nvidia_driver_major_version=560
 ```
 To see which driver versions are available, one can check the NVIDIA AL2023 [repository](https://developer.download.nvidia.com/compute/cuda/repos/amzn2023/).
+
+## Pause Container Image
+
+Since [PR #2000](https://github.com/awslabs/amazon-eks-ami/pull/2000), the pause
+container image is no longer pulled in at runtime on AL2023. `nodeadm` uses a
+static pause container reference to populate the
+[`sandbox`](https://github.com/containerd/containerd/blob/963c2160485631c3d4c1068aa28cea34c884a347/docs/cri/config.md?plain=1#L190)
+field of the containerd config.toml, which is baked into the AMI during the
+build based on the packer template variable `pause_container_image`.
+
+The default for `pause_container_image` is adequate for building in the `aws`
+partition, but if you require a region-specific ECR image, below is a table of
+the recommended pause container image for each region:
```

Comment:
Could we get by with only one region per partition? It'd be nice to only update this table when a new partition comes online.

Reply:
true, ill double check if that'll work

Reply:
i think the opt-in regions wont play nice with this, but i had another idea. what if we just reuse the EKS addon docs for this table?

https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html